data/tokens.json

```diff
@@ -3173,6 +3173,20 @@
       "value": "{Text.gtmds_color_text-color}",
       "type": "color"
     },
+    "onboarding": {
+      "gtmds_popover_color": {
+        "value": "{gtmds_color_gray-1}",
+        "type": "color"
+      },
+      "gtmds_popover_background-color": {
+        "value": "{Info.gtmds_color_info}",
+        "type": "color"
+      },
+      "gtmds_popover-footer_stepper_margin": {
+        "value": "auto",
+        "type": "spacing"
+      }
```

Comment:
I can't figure out where and how we use this in Figma so I left a comment there and let's use it to help me understand